experimental/agentic/src/main/java/io/serverlessworkflow/impl/expressions/agentic/AgenticModelFactory.java

```diff
@@ -45,57 +35,59 @@ public WorkflowModel fromAny(WorkflowModel prev, Object obj) {
 
   @Override
   public WorkflowModel combine(Map<String, WorkflowModel> workflowVariables) {
-    return new AgenticModel(workflowVariables, cognisphere);
+    throw new UnsupportedOperationException();
```

Comment:
This means no parallel execution. Are we sure?

Reply:
Fixed!